apps/src/templates/studentSnapshot/CodeWidget/CodeWidget.story.tsx

```diff
@@ -0,0 +1,132 @@
+import type {Meta, StoryObj} from '@storybook/react';
```

Comment:
I had to change a lot of things in the story since the fle structure is new. Therefore it almost looks like a new file.